packages/core/src/internal/services/global.service.ts

```diff
@@ -10,7 +10,7 @@ import { LogService } from './log.service.js';
 export class GlobalState {
   static get state(): Readonly<CDSState> {
     setupCDSGlobal();
-    return window?.CDS?.state;
+    return window?.CDS?.state as CDSState;
```

Comment:
This was throwing a error on next, this fixes the error or the global service PR will